detectron2/layers/csrc/box_iou_rotated/box_iou_rotated_utils.h

```diff
@@ -194,7 +194,7 @@ HOST_DEVICE_INLINE int convex_hull_graham(
     dist[i] = dot_2d<T>(q[i], q[i]);
   }
 
-#ifdef __CUDACC__
+#if defined(__CUDACC__) ||  __HIP_DEVICE_COMPILE__==1
```

Comment:
Hi @hgaspar , should we update here the same way as 6c216765478ad9eea97437b795fc294d1812c2c1?

Reply:
Thank you, yes, already done!